### DISCUSSION TITLE
XCSoar flight logs won't download onto WeGlide

### DISCUSSION THREAD

**Discussion by roscosoarer** [XCSoar/XCSoar#1404] [Q&A]
I have a flight log on my S23 that I can replay in XCSoar but I cannot upload  to WeGlide.  I think XCSoar logs are on S23 internal memory in an android folder that I cannot access or copy.  I have tried Phone Link but unsuccessful.  

What am I doing wrong?

**Comment by lordfolken** [comment]
The upload is automatic if configured in settings.

**Reply by mikelimaLS1** [reply]
Hello, I'm having the same problem. I've configured weglide in xcsoar but I don't understand how to transfer the files.

**Reply by lordfolken** [reply]
This unintuitive feature tries to upload the IGC file from a certified logger connected to XCSoar once you land.  
So basically if you have an IGC Flarm or Vario that is IGC certified, XCSoar will connect to it post landing, download the igc file and upload it to weglide.

**Reply by lordfolken** [reply]
WeGlide: Upload local igc file if no logger is connected #1778

**Reply by DanD222** [reply]
Is the upload only triggered on landing? That would make a big difference in usability vs a scenario of transferring the files post-landing.
